### PR TITLE
#7734 - missing descriptions in the manual

### PR DESCRIPTION
**Pull request by Vegebutcher**

### CODE REVIEW

File: G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookSaveAttachmentCommand.cs [hunk 1]
```diff
@@ -13,15 +13,15 @@
...
 namespace G1ANT.Addon.MSOffice
 {
-    [Command(Name = "outlook.saveattachment", Tooltip = "This command saves attachment as a local file.")]
+    [Command(Name = "outlook.saveattachment", Tooltip = "This command saves an attachment to a file")]
     public class OutlookSaveAttachmentCommand : Command
     {
         public class Arguments : CommandArguments
         {
-            [Argument(Required = true)]
+            [Argument(Name = "Attachment", Tooltip = "Email attachment to be saved", Required = true)]
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookSaveAttachmentCommand.cs]: it is correct of course but we never add `Name = ""` in the Argument attribute, the name of an argument is below so it's not needed

File: G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookSaveAttachmentCommand.cs [hunk 2]
```diff
@@ -13,15 +13,15 @@
...
             public OutlookAttachmentStructure Attachment { get; set; }
 
-            [Argument(Required = true)]
+            [Argument(Name = "Path", Tooltip = "Path to the saved file", Required = true)]
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookSaveAttachmentCommand.cs]: you can remove this argument here as well

File: G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookSaveAttachmentCommand.cs (hunk 1 above)
Comment (Vegebutcher) [reply G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookSaveAttachmentCommand.cs]:
```suggestion
            [Argument(Tooltip = "Email attachment to be saved", Required = true)]
```

File: G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookSaveAttachmentCommand.cs (hunk 2 above)
Comment (Vegebutcher) [reply G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookSaveAttachmentCommand.cs]:
```suggestion
            [Argument(Tooltip = "Path to the saved file", Required = true)]
```

**Pull request merged**